src/styles/reset/reset.scss

```diff
@@ -0,0 +1,113 @@
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
```

Comment:
remove to another file. It does not look like you resetting value here

Reply:
**Fixed**